docs/debugging.md

```diff
@@ -20,8 +20,8 @@ is compiled as part of building eBPF for Windows, as it is used in the unit test
 
              Section       Type  # Maps    Size
 ====================  =========  ======  ======
-               .text        xdp       1       3
-                 xdp        xdp       1      20
+               .text   xdp_test       1       3
```

Comment:
We should probably (in a separate issue and PR) update debugging.md to use a real program type that is in ebpf-for-windows rather than xdp.

Reply:
Filed issue #3088